src/reference/02-DetailTopics/01-Using-sbt/01-Command-Line-Reference.md

```diff
@@ -334,7 +334,7 @@ The following system properties can also be passed to `sbt`:
     <td><tt>sbt.color</tt></td>
     <td>String</td>
     <td><tt>auto</tt></td>
-    <td>Supercedes <tt>sbt.log.format</tt>.
+    <td>Supersedes <tt>sbt.log.format</tt>.
```

Comment:
Funny. Dale used to spell this word as superseed, I'd spell as supercede, and neither of us could spell!

https://ahdictionary.com/word/search.html?q=supersede&submit.x=0&submit.y=0 says

> The spelling with a _c_ has been in existence for 300 years and has traditionally been considered an error, but it appears so widely in books and other edited publications that this spelling must be considered standard.

Reply:
Normally I'm not normative.